### ISSUE TITLE
Week 5 announcements

### ISSUE THREAD

**Issue by joenash** [campus-experts/spring-2017#40]
## The TLDR this week:
**[Complete the mid-training survey](https://goo.gl/forms/wDBO77MzuKyiaAa72)**

- [Week 5 todo](https://github.com/campus-experts/spring-2017/blob/master/todos/week-5.md)
- [Last week's video](https://www.dropbox.com/s/8mk2lfcrvmj02cb/week4.mp4?dl=0), and last week's [tech talk](https://www.dropbox.com/s/nb6n9f9kr8nre1a/graphql.mp4?dl=0).
- Complete last week's exercises - [writing READMEs](https://github.com/campus-experts/spring-2017/blob/master/docs/write-a-readme.md)! Open an issue with your README review or link to your new README.
- This week we have two guest talks! TechNottingham on 26th April, and Mapping Applications tech talk on 27th April. 

## Welcome to Week 5 🚩

_Whoa-oh, we're halfway there_...
There are just 4 weeks worth of training ahead of us. With half of the training program completed, we want to make sure we're addressing your expectations and providing you the skills you need. Please consider filling out the mid-training survey; it helps us improve the program both for you, and future generations of participants.

This week we have two really exciting talks. First we'll be hearing from Emma and Andrew Seward of TechNottingham, a fantastic community, on just how they built it. It taps into a lot of topics we've discussed and definitely answers some of your questions. It's a great talk for improving your impact proposals!

We'll also be hearing about Geo and Mapping Applications from James Milner in our guest talk, more [here](https://github.com/campus-experts/spring-2017/issues/39).

The discussion topic this week is all about what happens to your community when you move on, whether that's graduating, relocating, or simply being too busy. [Check it out](https://github.com/campus-experts/discuss/issues/2)!

## This week on Slack
Missed a discussion on slack? Not to worry!

- Have you seen the GitHub Education forum? [Juan dropped in to help a poster](https://githubeducation.slack.com/archives/C2TQDRQSG/p1492693453357542)! 
- Gwen has a [really awesome reveal.js Git workshop](https://githubeducation.slack.com/archives/C2TQDRQSG/p1492549009319116)
- Are the talks hard for you to attend? [Let's get some new times.](https://githubeducation.slack.com/archives/C2TQDRQSG/p1492706463200387)
- A [heap of resources](https://githubeducation.slack.com/archives/C4PF3VCSU/p1492550629748580) for finding projects to contribute to.
